src/Sylius/Bundle/CoreBundle/Fixture/Factory/AddressExampleFactory.php

```diff
@@ -143,8 +140,6 @@ public function create(array $options = [])
             $address->setProvinceName($options['province_name']);
         }
 
-        $options['customer']->addAddress($address);
```

Comment:
Adding an address to the customer in address factory is not good as well, we should set a customer on the address instead.